docs/api/web-contents.md

```diff
@@ -375,6 +375,8 @@ win.webContents.on('will-prevent-unload', (event) => {
 })
 ```
 
+**Note:** This will be emitted for `BrowserViews` but will _not_ be respected - this is because we have chosen not to tie the `BrowserView` lifecycle to its owning BrowserWindow should one exist per the [specification](https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeunload_event).
```

Comment:
what about this event won't be respected? preventing the default?

Reply:
@nornagon yes - the function body will be run but the return value won't be respected

Reply:
Er, sorry for not getting back to this. I think this is unclear in the docs as written and should be clarified.